public/scripts/app.js

```diff
@@ -0,0 +1,47 @@
+(function(module) {
+
+  var Cars = function(instance) {
+    this.dt = instance.time;
+    this.img_url = instance.img_url;
+    this.lic_plate = instance.lic_plate;
+    this.lic_state = instance.lic_state;
+    this.make = instance.make;
+    this.model = instance.model;
+  };
+
+  Cars.all = [];
+
+  Cars.loadAll = function(data) {
+    Cars.all = [];
+    data.forEach(function(ele) {
+      Cars.all.push(new Cars(ele));
+    });
+  };
+
+  Cars.getData = function(ctx, next) {
```

Comment:
the local storage logic in this method doesn't make sense. you delete invaders from LS, then get the JSON file from PG, then set invaders to local storage, then get it right back and parse it for use in Cars.loadAll. As far as I can tell, you never utilize that LS item anywhere else, and it's redundant to pass it to Cars.loadAll when you already have the same thing in the data variable.